### PR TITLE
Metadata on the Paywall blog post

### PR DESCRIPTION
**Pull request by cnasc**
# Description

Now ready for a full review!

Screen Shot 2020-02-12 at 13 36 55-fullpage

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol.com/blog/metadata-collection.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Collecting Metadata for a Purchase
+subTitle: Connecting lock owners and key holders
+authorName: Christopher Nascone
+publishDate: February 1, 2020
```
Comment (cnasc) [inline unlock-protocol.com/blog/metadata-collection.md]: placeholder

File: unlock-protocol.com/blog/metadata-collection.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+description: Lock owners can require information from key purchasers before the transaction is inittiated.
+image: /static/images/blog/unlocking-smart-contracts/code.jpeg
```
Comment (cnasc) [inline unlock-protocol.com/blog/metadata-collection.md]: placeholder

File: unlock-protocol.com/blog/metadata-collection.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# Broader Availability of Metadata Collection
+
+Today we're proud to announce an extension to our embeddable Paywall application which allows creators to specify a number of form fields which should be presented to the purchaser before the purchase transaction begins. The technology behind this is our [metadata system](https://docs.unlock-protocol.com/developers/metadata) which we've already used behind-the-scenes in a few places. If you're already familiar with our Paywall, you can learn how to add metadata to your existing configuration [here](https://docs.unlock-protocol.com/applications/paywall/advanced-paywall-configuration). For everyone else, the remainder of this post is a worked example.
+
+# Collecting Information From Purchasers
```
Comment (julien51) [inline unlock-protocol.com/blog/metadata-collection.md]: Teaser... ;)

File: unlock-protocol.com/blog/metadata-collection.md (hunk 2 above)
Comment (cnasc) [reply unlock-protocol.com/blog/metadata-collection.md]: Just waiting on a graphic from @smombartz for this

**Comment by cnasc** [comment]
View in the list of posts:

image

**Pull request merged**